longshot/application/main.tf

```diff
@@ -0,0 +1,218 @@
+variable "pipeline" {}
+variable "pipeline_stage" {}
+variable "name" {}
+variable "host" {}
+
+variable "web_size" {
+  default = "Standard-1X"
+}
+
+variable "web_scale" {
+  default = "1"
+}
+
+variable "queue_size" {
+  default = "Standard-1X"
+}
+
+variable "queue_scale" {
+  default = "1"
```

Comment:
are all these defaults assume prod instance? maybe specify what defaults are good for. It might also be good to have separate templates with defaults for different environments: qa, dev, staging, prod.

Reply:
We'll likely use the same values here for all instances of this application.